micronota/parsers/embl.py

```diff
@@ -0,0 +1,432 @@
+r'''
+UniProtKB EMBL Parser
```

Comment:
certainly. it is missing writers, which is low priority in my list. so that gonna take a while.